CCS/3. Tutorials/5. 使用 TensorFlow Inception V3 訓練影像辨識模型、生成推論引擎.md

```diff
@@ -1 +1,154 @@
+
+```
+Image file type:     TensorFlow 18.12
+Image file version:  support Python 2
+Basic Configuration: model c.super 
+```
+
+### Step 3. Connect to the container, download the training program
+
+- See [Connection method](https://www.twcc.ai/doc?page=container#連線使用方式), and using Jupyter Notebook or SSH to connect to the default storage space of the container
+
+
+- Enter the following command to download the framework program of [TWCC GitHub Inception v3](https://github.com/TW-NCHC/AI-Services/tree/V3Training) image model training to the container
+
+ ```bash
+$ git clone https://github.com/TW-NCHC/AI-Services.git
+ ```
+
+
+  
+### Step 4. Conduct AI model training
+ 
+ - Enter the "**Tutorial_Three**" directory
+ 
+ ```bash
+ $ cd AI-Services/Tutorial_Three
+ ``` 
+ 
+ - Perform model training
+ 
+ ```bash
+$ bash V3_training.sh --path ./cifar-10-python.tar.gz
+  ``` 
+
+- The following message in terminal, indicating that the model is about to be trained
+
+:::warning
+![](https://cos.twcc.ai/SYS-MANUAL/uploads/upload_73e007a0bba2a9291a54bd03dd260893.png)
+:::
+
+ 
+- During the training, you can view the resource usage of CPU/GPU, memory, and network on the "**Interactive  Container Details**" page
+
+![](https://cos.twcc.ai/SYS-MANUAL/uploads/upload_ac448244502db32c89da844e30e525bf.png)
+
+
+ 
+- After the model is trained, it will be stored in the path: `AI-Services/Tutorial_Three/inceptionv3/train/weights` 
+
+:::warning
+![](https://cos.twcc.ai/SYS-MANUAL/uploads/upload_2a714f02d570256125e5b2ab9c92234c.png)
+:::
+
+
+## Part 2. Generating inference recognition engine
+
+The following tutorial demonstrates how to use the trained model to generate an inference engine and provide web services for image inference and identification.
+
+### Step 1. Connect to container
+
+See [Connection method](https://www.twcc.ai/doc?page=container#連線使用方式).
+
+    
+### Step 2. Generating inference recognition engine 
+ 
+- Enter the Tutorial_Three directory
+
+```bash
+$ cd AI-Services/Tutorial_Three
+``` 
+ 
+- Activate the AI inference engine service
+
+```bash
+$ bash V3_inference.sh
+``` 
+ 
+![](https://cos.twcc.ai/SYS-MANUAL/uploads/upload_309b6636ffb075f5de28ceeca0b7bac9.png)
+
+
+### Step 3. Image recognition website
+
+- Please go to the detailed information page of the Interactive Container, first:arrow_right: click on the "**Associate**" service port :arrow_right: check on "**5001**" :arrow_right: click OK to open the HTTP web service endpoint
+==Picture Need to update:exclamation:==
+![](https://cos.twcc.ai/SYS-MANUAL/uploads/upload_fd0f4028883bfaca0f24e08a3eaf4012.png)
+
+
+- Open the web browser and enter **`Container IP: Public Port`** in the address bar, according to the example is `203.xxx.xxx.xxx:50962`, you can starts to use the AI Inference service.
```

Comment:
[SUG]

, according to the example is `203.xxx.xxx.xxx:50962` > (as shown below on the Interactive Container Details page)

Reply:
Inference > inference